### ISSUE TITLE
"Logged in" dropdown disappears on mobile and 200% zoom

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/heracles-etd#247]
On narrow viewports, and at 200% zoom, the "Logged in" dropdown disappears.

### Desktop 100% zoom
<img width="1504" height="509" alt="Image" src="https://github.com/user-attachments/assets/ef3efedb-051d-42af-a8a4-baa879f8d1da" />

### Desktop 200% zoom
<img width="1495" height="771" alt="Image" src="https://github.com/user-attachments/assets/e8d9ae80-196e-4ccc-abc7-7f637152c931" />

### Mobile
<img width="400" height="708" alt="Image" src="https://github.com/user-attachments/assets/f49c02b2-5476-4995-8c0a-e3f514e9c133" />

### Component library
The component library addresses this by placing the dropdown in a hamburger menu. See: https://sul-dlss.github.io/component-library/header/.

<img width="774" height="186" alt="Image" src="https://github.com/user-attachments/assets/04a9f6dc-240b-46be-8fd3-2405c10ba22d" />

### Reported by SODA
> 
> [Summary]
> Functionality is lost at 200% zoom
> 
> [Description]
> "Functionality is lost when the page is zoomed to 200%.
> At 200% zoom, the option to logout is missing. "
> 
> [Impact]
> Serious
> 
> [Recommended to fix]
> Ensure that all content is available when zoomed into 200%. 
>
